tests/safety/test_toyota.py

```diff
@@ -117,6 +124,59 @@ def test_accel_actuation_limits(self, stock_longitudinal=False):
             should_tx = np.isclose(accel, 0, atol=0.0001)
           self.assertEqual(should_tx, self._tx(self._accel_msg(accel)))
 
+  def test_steer_req_bit_frames(self):
```

Comment:
move to generic tests and check if `MIN_VALID_STEERING_FRAMES` is set. the test case should run for all safety modes, e.g. on hyundai (for now), it will enforce that glitching is not allowed